docs/source/getting-started/index.adoc

```diff
@@ -5,8 +5,8 @@
 This section contains information about installing Minishift, quickstart steps, and sample
```

Comment:
Sure, I just realized now that we never settled on this, but the openshift folks said it doesn't matter to publishing so I can fix it here. We should also add this to our own doc guidelines probably, no?